generator/config/accumulator/percentile.yaml

```diff
@@ -25,7 +25,7 @@ arguments:
     -
         name: p
         type:
-            - resolvesToArray # of resolvesToNumber
```

Comment:
I was keeping this comments as a note for later, as we would like to extend type definition for generic lists and maps.

Reply:
Yeah, that makes sense - same as above, those got removed due to the yaml parser not preserving comments, but will try and find a way to avoid that.